src/angular/input/input.ts

```diff
@@ -66,7 +66,8 @@ const _SbbInputBase = mixinErrorState(
     // state usually overlaps with `aria-required` when the input is empty and can be redundant.
     '[attr.aria-invalid]': '(empty && required) ? null : errorState',
     '[attr.aria-required]': 'required',
-    '[attr.placeholder]': 'placeholder || null',
+    '[attr.placeholder]': `!readonly ? placeholder || null : '-'`,
```

Comment:
```suggestion
    '[attr.placeholder]': `!readonly ? (placeholder || null) : '-'`,
```
This is easier to read to me. What do you think?